README.md

```diff
@@ -115,7 +115,13 @@ yarn test:unit src/applications/path/to/tests/**/*.unit.spec.js*
 ```
 
 ### Browser tests
-To **run all browser tests**, you first need two things:
+To **run all browser tests**, you first need three things:
+1. Install the Java JDK on MacOS:
+    ```
+    brew update
+    brew tap adoptopenjdk/openjdk
+    brew cask install adoptopenjdk8
+    ```
```

Comment:
I don't think this is necessary, but I'm on Linux. cc @rianfowler

Reply:
Java is NOT installed on MacOS by default. I think it used to be once upon a time. (?)

Reply:
https://www.forbes.com/sites/adriankingsleyhughes/2012/10/19/apple-makes-os-x-safer-by-removing-java/

Reply:
Talked with some other folks on the front end team and it sounds like I'm wrong. :+1: 

I like your idea (in Slack) about adding the "if you don't already have it" bit, but it's probably fine as-is.